TickSpec/ScenarioGen.fs

```diff
@@ -348,13 +348,7 @@ let defineStepMethod
     // Emit arguments
     let ps = mi.GetParameters()
     let zipped = Seq.zip args ps
-#if SILVERLIGHT
-    // For instance methods get instance value from service provider

```

Comment:
IME it can be helpful to have these variations about when porting to more constrained platforms, i.e. .NET Standard. Arguably the merging / removal of this could should wait until that point (only being devil's advocate really though)